<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Front Panel Controller</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../tapbarresource.qrc">
    <normaloff>:/root/images/Images/ECG-img.png</normaloff>:/root/images/Images/ECG-img.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>7</number>
      </property>
      <widget class="QWidget" name="ECGBASE">
       <attribute name="title">
        <string>ECG Baseline</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>691</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="loECGBaseline"/>
       </widget>
      </widget>
      <widget class="QWidget" name="RPEAKS">
       <attribute name="title">
        <string>R-peaks</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>641</width>
          <height>251</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="loRpeaks"/>
       </widget>
      </widget>
      <widget class="QWidget" name="HRV1">
       <attribute name="title">
        <string>HRV 1</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>591</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="lo_HRV1"/>
       </widget>
      </widget>
      <widget class="QWidget" name="HRV2">
       <attribute name="title">
        <string>HRV 2</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>591</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="lo_HRV2"/>
       </widget>
      </widget>
      <widget class="QWidget" name="HRF_DFA">
       <attribute name="title">
        <string>HRV DFA</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>681</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="loHRVdfa"/>
       </widget>
      </widget>
      <widget class="QWidget" name="ST_SEGMENT">
       <attribute name="title">
        <string>ST Segment</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>601</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="loSTSegment"/>
       </widget>
      </widget>
      <widget class="QWidget" name="HEART_CLASS">
       <attribute name="title">
        <string>Heart Class</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="loHeartClass"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="T_WAVE_ALT">
       <attribute name="title">
        <string>T Wave Alt</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="loTAltClass"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../tapbarresource.qrc"/>
 </resources>
 <connections/>
</ui>
